### DISCUSSION TITLE
Loading old save file from February no longer working in browser or mac app

### DISCUSSION THREAD

**Discussion by Johnnydoc** [Lurkars/gloomhavensecretariat#549] [Q&A]
My wife and I have been playing the Frosthaven campaign for a number of months. We last played February 7th, 2024 with a .json back up from that day on our computer. When I loaded up the browser today, instead of pre-populating the campaign as in the past, it was back to the main screen of the browser asking to start a new campaign. Going to the 'import campaign game' allows me to click on the .json file but nothing happens or changes on the browser. The only thing that seems to have worked on loading is a data dump from November 2023 :( eek!! I tried to download an older version of the app for mac to see if that would allow loading but it did not. Please help!! Any suggestions on how to to get our saved game working??

**Comment by Lurkars** [comment]
Can you please send me the backup file to issue@gloomhaven-secretariat.de? It's mostly migration issues, I can quickly fix, but want able to test beforehand.
